src/value.rs

```diff
@@ -1534,10 +1527,10 @@ impl<'a> Set<'a> {
 
 impl<'a> Function<'a> {
     /// Create a function in the current execution context for a given callback.
-    pub fn new(isolate: &'a isolate::Isolate,
+    pub fn new<'b>(isolate: &'a isolate::Isolate,
```

Comment:
Ah, yeah that is a problem. The issue I was trying to solve was if you have a module level function instead of a closure and try to pass the reference to it to Function::new it errors with `borrowed value does not live long enough` because the reference only lives for that line. Is there some other way to handle the lifetime so module level functions can just be passed to this call?